### PR TITLE
feat: project setup and onboarding screens

### PR DESCRIPTION
**Pull request by maxpetretta**
- Reconfigures the project
- Deprecates old assets and scenes
- Refactors `common` into `components`, `services`, `lib`, and `hooks`
- Switches from app router to pages
- Adds new identity storage management hook
- Adds onboarding and signature confirmation screens

Should be the last messy PR 🙈

### CODE REVIEW

File: tailwind.config.js [hunk 1]
```diff
@@ -55,13 +55,15 @@ module.exports = {
     colors: {
       ...mirrorHexColors([
         "#000000",
+        "#00c313",
```
Comment (igorosip0v) [inline tailwind.config.js:58]: We should start naming colors as it is in Figma, I guess.

**Review by igorosip0v — APPROVED** [review]
We should remove Version 2.0.0 at the bottom I guess.
And also fix background of the colored icons. (But this change also exists in @michaelfurutin local branch afaik)

File: tailwind.config.js (hunk 1 above)
Comment (maxpetretta) [reply tailwind.config.js:58]: Will fix in future PR

**Comment by maxpetretta** [comment]
> We should remove Version 2.0.0 at the bottom I guess. And also fix background of the colored icons. (But this change also exists in @michaelfurutin local branch afaik)

Space feels empty on mobile w/o the version, so going to leave for now. Will let Michael handle the icons

**Pull request merged**